client/client.go

```diff
@@ -269,6 +277,9 @@ func (rq *defaultRequester) dispatch(ctx context.Context, method, urlpath string
 	if rq.userAgent != "" {
 		req.Header.Set("User-Agent", rq.userAgent)
 	}
+	if rq.contentType != "" {
+		req.Header.Set("Content-Type", rq.contentType)
+	}
```

Comment:
So per above, I think we can just set this always:
```suggestion
	req.Header.Set("Content-Type", "application/json")
```